src/errors.ts

```diff
@@ -112,3 +116,43 @@ export enum ValidationErrorCodes {
     InternalError = 1008,
     TokenNotSupported = 1009,
 }
+
+export abstract class AlertError {
+    public abstract message: string;
+    public shouldAlert: boolean = true;
+}
+
+export class ExpiredOrderError extends AlertError {
+    public message = `Found expired order! `;
```

Comment:
```suggestion
    public message = `Found expired order!`;
```

Reply:
Ah good catch, thanks. this was leftover from when I was concatenating the details into the same string!